desk/app/activity.hoon

```diff
@@ -1,6 +1,8 @@
 ::
 /-  a=activity
 /+  default-agent, verb, dbug
+::  build for performance
+/+  activity-json
```

Comment:
Isn't this already in cache by virtue of this same import in the marks, which are being warmed?